README.rst

```diff
@@ -213,6 +213,7 @@ Scrapers available for:
 - `https://foodrepublic.com/ <https://foodrepublic.com>`_
 - `https://www.forksoverknives.com/ <https://www.forksoverknives.com/>`_
 - `https://forktospoon.com/ <https://forktospoon.com/>`_
+- `https://fortyaprons.com/ <https://40aprons.com/>`_
```

Comment:
The domain of this site is **40**aprons rather than **forty**aprons so it should be closer to line 81